jms-quickstart/src/main/java/org/acme/jms/PriceConsumer.java

```diff
@@ -52,6 +52,7 @@ public void run() {
                     return;
                 }
                 lastPrice = message.getBody(String.class);
+                System.out.println("Got last price: " + lastPrice);
```

Comment:
Use log.info?

Reply:
There are no logger. We already have System.out in the test resource, so discarded the use of it.
Hopefully, once I understand the issue, this will be reverted.